City-of-Helsinki/helsinki-paatokset
# PP-0: Misc decision fixes

**To test**
- Read the cron job scripts just to make sure the changes make sense. The timings have been changed because there seemed to be some database overload issues when a lot of the migrations and aggregations ran at the same time.
- Checkout branch, run `make drush-cr`
- Run `make ahjo-migrations` if you don't have Ahjo data yet
- Go to: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginhallitus/asiakirjat
  - There should now be a margin between the "Select year" heading and the year tabs
  - There should now be a margin between the document icon and the  "Read decision announement" label
- Go here: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginhallitus/asiakirjat/0040020221#paatostiedote
  - The "Kokousaika:" line should be bolded
  - The font sizes for the agenda list and decision meta text (date and location) should be 18px.
